lakey_service/downloader/executors/databricks.py

```diff
@@ -1,13 +1,43 @@
+from databricks_api import DatabricksAPI
+from datetime import datetime
+from django.conf import settings
 
+from .baseexecutor import BaseExecutor
 
-class DataBricksExecutor:
-    # https://docs.databricks.com/api/latest/jobs.html#jobsnotebooktask
 
-    def start_cluster(self):
-        pass
+class DatabricksExecutor(BaseExecutor):
 
-    def start_job(self, spec):
-        pass
+    def __init__(self):
+        self.db = DatabricksAPI(
+            host=settings.DATABRICKS_HOST,
+            token=settings.DATABRICKS_TOKEN
+        )
 
-    def get_job_status(self, job_id):
-        pass
+    def __generate_file_path(self):
+        """ Return path of output file """
+        now = datetime.now()
+        output_location = settings.DATABRICKS_RESULTS_LOCATION
+        filename = now.strftime("%Y%m%d-%H%M%S")
+
+        return output_location+filename+"_output.csv"
```

Comment:
for the path constructions we should always use `os.path.join`
```python
filename = f'{now.strftime("%Y%m%d-%H%M%S")}_output.csv'
return os.path.join(settings.DATABRICKS_RESULTS_LOCATION, filename)
```

Reply:
if you're asking yourself a question what's wrong with using simple string concatenate for the path construction, well the problem is with slashes which are nicely handled by the `os.path.join`